lib/game_statistics.rb

```diff
@@ -0,0 +1,72 @@
+
+module Game_statistics
```

Comment:
modules should follow camel case naming conventions, i.e, GameStatistics

Reply:
changed module name and name in required sections